src/Build/BackEnd/Components/RequestBuilder/IntrinsicTasks/ItemGroupLoggingHelper.cs

```diff
@@ -138,7 +130,7 @@ internal static string GetStringFromParameterValue(object parameterValue)
         [ThreadStatic]
         private static List<KeyValuePair<string, string>> keyValuePairList;
 
-        private static void AppendStringFromParameterValue(ReuseableStringBuilder sb, object parameterValue)
+        private static void AppendStringFromParameterValue(ReuseableStringBuilder sb, object parameterValue, bool logItemMetadata = true)
```

Comment:
Might it be better to not log item metadata by default?

Reply:
I think we need item metadata for .binlog to be usable. From experience it contains tons of valuable information, such as Private, CopyToOutput, which path a reference was resolved from, etc.